test/python/dagcircuit/test_dagcircuit.py

```diff
@@ -10,6 +10,8 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
+# pylint: disable=invalid-name
```

Comment:
What's the invalid name that pylint was complaining about?

Reply:
This is, I believe, because of the additional letter-only names I used. Like "c" and "d".

There were already letter-only names, so I figured this should be allowed. Let me know if you have other suggestions :)